io_scene_dae/export_dae.py

```diff
@@ -61,27 +61,26 @@ def snap_tup(tup):
 
 
 def strmtx(mtx):
-    s = " "
+    s = ""
     for x in range(4):
         for y in range(4):
-            s += str(mtx[x][y])
-            s += " "
-    s += " "
+            s += " {}".format(mtx[x][y])
+    s = " {} ".format(s)
     return s
```

Comment:
That's easily fixed by replacing `s += " {}".format(mtx[x][y])` with `s += "{} ".format(mtx[x][y])`, but again I don't know of it's of any relevance for the plugin.